tests/pytest/test_array.py

```diff
@@ -164,30 +160,13 @@ def test_slice_correct_infered():
 def test_array_concat():
     l = Array("01ZX", Range(0, 'to', 3))
```

Comment:
```suggestion
    """Test array concatenation (``@``)."""
    l = Array("01ZX", Range(0, 'to', 3))
```

Reply:
Should I add docstrings to all the tests, or just this one?

Reply:
Just this one, the other ones are self-explanatory IMO; I just thought that every reminder that `@` means "concatenation" is useful. (Reminders will become less important once `@` is showing up in examples, so feel free to ignore the docstring suggestion.)